schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+                        "type": "string"
+                    },
+                    "familyName": {
+                        "description": "The family name of the creator.",
+                        "type": "string"
+                    },
+                    "affiliation": {
+                        "$ref": "#/definitions/affiliations"
+                    }
+                }
+            },
+            "minItems": 1,
+            "uniqueItems": true
+        },
+        "publisher": {
+            "description": "The publisher of the resource.",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "The identifier for the publisher.",
+                    "type": "string"
+                },
+                "name": {
+                    "description": "The name of the publisher.",
+                    "type": "string"
+                }
+            },
+            "required": [
+                "name"
+            ]
+        },
+        "date": {
+            "description": "The dates for the resource.",
+            "type": "object",
+            "format": "date"
+        },
+        "titles": {
+            "description": "The titles of the resource.",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "title": {
+                        "description": "The title of the resource.",
+                        "type": "string"
+                    },
+                    "type": {
+                        "description": "The type of the title.",
+                        "type": "string",
+                        "enum": [
+                            "AlternativeTitle",
+                            "Subtitle",
+                            "TranslatedTitle"
+                        ]
+                    }
+                },
+                "required": [
+                    "title"
+                ]
+            }
+        },
+        "container": {
+            "description": "The container of the resource.",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "The identifier for the container.",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "The type of the container.",
+                    "type": "string",
+                    "enum": [
+                        "Book",
+                        "BookSeries",
+                        "Journal",
+                        "Periodical",
+                        "ProceedingsSeries",
+                        "Series"
+                    ]
+                },
+                "title": {
+                    "description": "The title of the container.",
+                    "type": "string"
+                }
+            }
+        },
```

Comment:
This isn't a set of fields that we have looked at in our use cases so far, but I'm wondering if they could be captured using the `related_identifiers` field (at least in the KBCMS) using the part of relationship type.

Reply:
Like everything in commonmeta, this is evolving until version 1.0. The latest version of commonmeta for example added `DataCatalog` and `Repository`. Container is a central concept in many metadata schemata, e.g. csl, crossref or bibtex, and my past experience with using `related_identifers` was that it complicated things too much.

For data repositories I like `DataCatalog` and the re3data URL as `id`.

Reply:
IIRC I've also seen examples where a DOI'd dataset acts as a container for other DOI'd datasets.

Reply:
Yes, that is a common use case, the HasPart/IsPartOf relationship. Will think more how this can best be described.